### DISCUSSION TITLE
如何正确地发送戳一戳CQCode

### DISCUSSION THREAD

**Discussion by DreamPrism** [Hoshikawa-Kaguya/Sora#77] [Q&A]
如题，为什么用SoraSegment.Poke生成的戳一戳CQ码在构造MessageBody的时候会变成null啊
![image](https://user-images.githubusercontent.com/52944979/200452308-1bcc8184-a2be-4084-899f-e9997e01a40a.png)

**Comment by Yukari316** [comment]
无法复现
请检查gocq的上报格式设置

测试代码
```csharp
[SoraCommand(CommandExpressions = new[] { "1" },
                 SourceType = SourceFlag.Group,
                 Priority = 0)]
    public static async ValueTask TestCommand1(GroupMessageEventArgs eventArgs)
    {
        eventArgs.IsContinueEventChain = false;
        MessageBody message = new MessageBody
        {
            SoraSegment.Poke(eventArgs.Sender)
        };
        await eventArgs.Reply(message);
    }
```
log:
![image](https://user-images.githubusercontent.com/7535224/200545786-2e2a06a5-1b8a-429c-8eed-1380e6547608.png)
测试结果:
![image](https://user-images.githubusercontent.com/7535224/200545914-8ae4cf13-2628-478c-be46-8eb19d8de39e.jpg)
